RecoTracker/IterativeTracking/python/LowPtTripletStep_cff.py

```diff
@@ -199,28 +210,38 @@
     TrajectoryBuilderPSet = cms.PSet(refToPSet_ = cms.string('lowPtTripletStepTrajectoryBuilder')),
     clustersToSkip = cms.InputTag('lowPtTripletStepClusters'),
     doSeedingRegionRebuilding = True,
-    useHitsSplitting = True
+    useHitsSplitting = True,
+    TrajectoryCleaner = 'lowPtTripletStepTrajectoryCleanerBySharedHits'
```

Comment:
I had originally moved it because it seemed innocuous. In fact, my change has no affect on `trackingPhase2PU140`. Before my change, `trackingPhase2PU140` would have modified `clustersToSkip` and `phase2clustersToSkip` but left `TrajectoryCleaner`. This era would still use the same python object instance for `lowPtTripletStepTrackCandidates` as the non-era version. Then after `trackingPhase2PU140` was changed, the call to 
```python
lowPtTripletStepTrackCandidates.TrajectoryCleaner = 'lowPtTripletStepTrajectoryCleanerBySharedHits'
```
would change `lowPtTripletStepTrackCandidates` both in the era and non-era case. Therefore moving setting of `TrajectoryCleaner` into the `clone` should make no functional difference.

Reply:
I also had to move setting of `lowPtTripletStepTrackCandidates.TrajectoryCleaner` to before the `fastSim` era change since the resulting `lowPtTripletStepTrackCandidates` object after the `fastSim` is applied is a module that does not have the parameter `TrajectoryCleaner`.